src/System.Linq.Parallel/src/System/Linq/Parallel/Channels/AsynchronousChannel.cs

```diff
@@ -607,7 +613,7 @@ private T[] InternalDequeueChunk()
 
             // Zero out contents to avoid holding on to memory for longer than necessary. This
             // ensures the entire chunk is eligible for GC sooner. (More important for big chunks.)
-            _buffer[consumerBufferIndex] = null;
+            _buffer[consumerBufferIndex] = null!;
```

Comment:
`_buffer` can contain null arrays.  It should instead be defined as `T[]?[]`.

Reply:
Banged here because it was nulled after used, anyways better to be declared as can contain null